client/src/components/Auth/store/Auth.store.ts

```diff
@@ -235,7 +236,16 @@ export class AuthStore extends AbstractStore {
                             this.client.avatarUrl = avatarUrl
                         })
                     })
-                    .catch()
+                    .catch((err) => {
```

Comment:
return the service call from .then and remove this .catch

Reply:
ok